core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -132,10 +132,10 @@ angular.module('oppia').directive('storyNodeEditor', [
                 $scope.skillInfoHasLoaded = true;
                 $rootScope.$applyAsync();
               });
-            for (var idx in skillSummaries) {
+            skillSummaries.forEach(idx => {
               $scope.skillIdToSummaryMap[skillSummaries[idx].id] =
-                skillSummaries[idx].description;
-            }
+                skillSummaries[idx].description;              
+            }); 
```

Comment:
`skillSummaries` is a list. Does `forEach` give 0-indexed indices? Or does it give an element of the list?

Reply:
Oops! correcting it